docs/breaking-changes.md

```diff
@@ -14,6 +14,11 @@ This document uses the following convention to categorize breaking changes:
 
 ## Planned Breaking API Changes (23.0)
 
+### Removed: Windows 7 and 8 support
+
+Windows 7 and 8 are not supported anymore as Chromium 110 dropped support:
+https://support.google.com/chrome/thread/185534985/sunsetting-support-for-windows-7-8-8-1-in-early-2023?hl=en
+
```

Comment:
```suggestion
### Removed: Windows 7 / 8 / 8.1 support

Windows 7, Windows 8, and Windows 8.1 are not supported anymore as Chromium 110 dropped support:
https://support.google.com/chrome/thread/185534985/sunsetting-support-for-windows-7-8-8-1-in-early-2023?hl=en

Older versions of Electron will continue to run on these operating systems, but Windows 10 or later will be required to run Electron v23.0.0 and higher.

```

I think we should add more details to the text to align with what Chrome is saying in their post. Also highlight 8.1 since I think it's technically a different OS from Windows 8.